Sample Project/SimpleLineChart/ViewController.m

```diff
@@ -53,6 +53,10 @@ - (void)viewDidLoad {
     self.myGraph.enableReferenceYAxisLines = YES;
     self.myGraph.enableReferenceAxisFrame = YES;
     
+    //line widths
+    self.myGraph.widthLine = 4;
+    self.myGraph.widthReferenceLines = 1;
```

Comment:
If I'm correct, these properties can be set in Interface Builder and do not need to be demonstrated here in the implementation. Could you remove lines 56-59 and make the appropriate changes in IB?